electron-builder.yaml

```diff
@@ -2,7 +2,7 @@ directories:
   buildResources: buildResources
   output: dist
 appId: edu.mit.scratch.scratch-desktop
-productName: "Scratch Desktop"
+productName: "Scratch 3"
```

Comment:
The electron-builder `productName` is used for packaging purposes, such as the name of the application file on disk.